doc/yeast-uniformity.md

```diff
@@ -0,0 +1,86 @@
+# Yeast uniformity test
+
+Version 0.2.0
+
+## Overview
+
+This test is performed on stored slants with yeast to increase certainty that population is indeed monoclonal and samplings from this storage unit are indeed genetically identical.
+
+## Prior knowledge
+
+Prior probability that culture is monoclonal: Pmp = 99%
+
+Mutation rate: km = 1e-7/s (P(mutation>1 | t) = 1 - exp(-t * km))
```

Comment:
I am not sure how to read this line.

Reply:
this is probability of mutations count being non-zero at time t